templates/zerver/landing_nav.html

```diff
@@ -7,6 +7,7 @@
 {% endif %}
 
 {% if not isolated_page %}
+<!-- We have separate copies of the navbar list for web and mobile. Please update both the versions when making changes. -->
 <nav class="top-menu">
```

Comment:
Is there already a similar comment at the other copy too?

In general for this sort of thing it's important to have a copy at each end, pointing to the other — otherwise it's easy to edit one side and not notice that the other exists.

Reply:
Missed to add it. Pushed PR to add there too. Thanks!